ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.13.yaml

```
@@ -84,41 +58,70 @@ tests:
         else
           OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}-index;
         fi;
-        make -k VERBOSE=2 TEST_ACCEPTANCE_START_SBO=scenarios OPERATOR_INDEX_IMAGE_REF=$OPERATOR_INDEX_IMAGE_REF CATSRC_NAME=sbo-pr-checks SKIP_REGISTRY_LOGIN=true EXTRA_BEHAVE_ARGS="-k --tags=@upgrade-with-olm --tags=~@disable-openshift-4.9 --tags=~@disable-openshift-4.8+ --tags=~@disable-openshift-4.9+" -o registry-login test-acceptance test-acceptance-artifacts
+        make -k VERBOSE=2 TEST_ACCEPTANCE_START_SBO=scenarios OPERATOR_INDEX_IMAGE_REF=$OPERATOR_INDEX_IMAGE_REF CATSRC_NAME=sbo-pr-checks SKIP_REGISTRY_LOGIN=true EXTRA_BEHAVE_ARGS="--tags=@upgrade-with-olm --tags=~@disable-openshift-4.13 --tags=~@disable-openshift-4.10+ --tags=~@disable-openshift-4.11+ --tags=~@disable-openshift-4.12+ --tags=~@disable-openshift-4.13+" -o registry-login test-acceptance test-acceptance-artifacts
       from: src
       resources:
         requests:
           cpu: 100m
     workflow: ipi-aws
 - as: validate-master-build
-  cron: 52 19 1,18 * *
+  cron: 0 0 * * *
   steps:
     cluster_profile: aws
-    post:
-    - as: gather-must-gather
+    test:
+    - as: test
+      cli: latest
       commands: |
-        curl -skSL https://raw.githubusercontent.com/openshift/release/master/ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh | /bin/bash -s
-      from: cli
-      optional_on_success: true
+        VERBOSE=2 make -k EXTRA_BEHAVE_ARGS="--tags=~@crdv1beta1 --tags=~@olm-descriptors --tags=~@upgrade-with-olm --tags=~@disable-openshift-4.12 --tags=~@disable-openshift-4.13 --tags=~@disable-openshift-4.10+ --tags=~@disable-openshift-4.11+ --tags=~@disable-openshift-4.12+ --tags=~@disable-openshift-4.13+" test-acceptance-with-bundle test-acceptance-artifacts
```

```suggestion
        VERBOSE=2 make -k EXTRA_BEHAVE_ARGS="--tags=~@crdv1beta1 --tags=~@olm-descriptors --tags=~@upgrade-with-olm --tags=~@disable-openshift-4.13 --tags=~@disable-openshift-4.10+ --tags=~@disable-openshift-4.11+ --tags=~@disable-openshift-4.12+ --tags=~@disable-openshift-4.13+" test-acceptance-with-bundle test-acceptance-artifacts
```

As it is 4.13 pipeline